### ISSUE TITLE
GLPI 9.2 released

### ISSUE THREAD

**Issue by vollkommenIrrelevant** [pluginsGLPI/datainjection#47]
Before the installation the plugin "This plugin requires GLPI 0.85 or higher". Has anyone already checked whether it works without problems?

**Comment by trasher** [comment]
This plugin is not compatible with GLPI 9.2

**Comment by vollkommenIrrelevant** [comment]
Reopen, no solution for GLPI 9.2...

**Comment by trasher** [comment]
As you wish... PR on 9.2 compatibility are welcome.

**Comment by trasher** [comment]
No, the plugin is **not** compatible with 9.2. Changing the value as you advice will not fix the issues.

This is maybe working, but only because of you specific usage.

**Comment by samww13** [comment]
Yes I confirm plugin doesn't work on 9.2  ;-)

**Comment by MichalBerta** [comment]
I also confirm it is not possible to install in 9.2 and hack from above is not working

**Comment by yoshi314** [comment]
This is the only thing preventing me from 9.2 upgrade. I guess i'll have to learn php.

**Comment by trasher** [comment]
If you do not use plugin to work on operating systems; you can try the `feature/9.2-compat` branch on the repo; this one should work with GLPI 9.2. If you want to use operating systems, you'll have to wait for now.

**Comment by barbuslex** [comment]
+1

**Comment by sector7gp** [comment]
feature/9.2-compat +1

**Comment by huornlmj** [comment]
Where can one get the "feature/9.2-compat" version of the plugin for 9.2?

**Comment by yoshi314** [comment]
@huornlmj just checkout the branch of that name

**Comment by toucom** [comment]
works with 9.1.6 version or lower

**Comment by kedson42** [comment]
@trasher , what needs to be adjusted on the plugin so it can work on GLPI 9.2?

Maybe if you give any tips the dev team here can help you out.

**Comment by trasher** [comment]
@kedson42 the Operating System part that has changed on GLPI, mainly.

**Comment by kedson42** [comment]
I see
In our case, we want to import a CSV with a service catalogue. Do any of you guys know another solution for that?

**Comment by vollkommenIrrelevant** [comment]
The best would be an adaptation of the plugin. Did you try to install the plugin and import the data?
To install you have to adjust version_compare in setup.php.
